prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,38 @@
+const number1 = 3;
+const number2 = 4;
+const product = number1 * number2;
+console.log('value of product:', product);
+console.log('typeof product:', typeof product);
+
+const charge = 6;
+const payment = 8;
+const amountRemaining = charge - payment;
+console.log('value of amountRemaining:', amountRemaining);
+console.log('typeof amountRemaining:', typeof amountRemaining);
+
+const tests = 7;
+const assignments = 5;
+const final = 9;
+const grade = (tests + assignments + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+const firstName = 'Cindy';
+const lastName = 'Le';
+const fullName = firstName + ' ' + lastName;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+const pH = 2;
+const isAcidic = pH < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+const numberOfSoldiers = 235;
+const isSparta = numberOfSoldiers === 300;
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+const nickname = fullName + ' is the GOAT';
```

Comment:
you want to define this one with let, and then reassign the new value to it as noted in the instructions.